Support remote installation of modules in private registries (#688)
kyma-project/lifecycle-manager: docs/developer-tutorials/config-private-registry.md

   > **NOTE:** The `"operator.kyma-project.io/managed-by": "lifecycle-manager"` label is mandatory for the Lifecycle Manager runtime controller to know which resources to cache.

> 3. Deploy the secret in the same cluster where the ModuleTemplate is to be located i.e if the ModuleTemplate is in the SKR cluster, then the secret should be deployed to the SKR, otherwise it should be deployed to the KCP cluster.

Reviewer: ```suggestion
3. Deploy the Secret in the same cluster where the ModuleTemplate is to be located. For example, if the ModuleTemplate is in the SKR cluster, then the Secret should be deployed to the SKR, otherwise, it sould be deployed to the KCP cluster.
```

> 3. Deploy the Secret in the same cluster where the ModuleTemplate is to be located. For example, if the ModuleTemplate is in the SKR cluster, then the Secret should be deployed to the SKR, otherwise, it should be deployed to the KCP cluster.

Reviewer: ```suggestion
3. Deploy the Secret on the same cluster where the ModuleTemplate CR is to be located. For example, if the ModuleTemplate CR is on the SKR cluster, then the Secret should be deployed to the SKR, otherwise, it should be deployed to the KCP cluster.
```
